### DISCUSSION TITLE
Setting the window title according to the connection name/ID

### DISCUSSION THREAD

**Discussion by paulsommer** [rustdesk/rustdesk#4464] [Feature Request]
Hi,

I am using the Linux Version of the client.

I would love the window title to reflect the connection that this window is on.
Should be easy to be done (setting the window title)
When I have several connections open it is impossible to tell one from the other in the taskbar.

Even cooler:
Open several connections in tabs instead of windows.
Possibly more difficult.

Cheers, Paul

**Comment by rustdesk** [comment]
[nightly ](https://github.com/rustdesk/rustdesk/releases/tag/nightly)

**Comment by paulsommer** [comment]
Yes, got it. It's great.
